docs/dev/guidelines/server-tests.rst

```diff
@@ -219,7 +219,7 @@ the name of the test group (matching the ``abstract`` base class name) as a para
     @Execution(ExecutionMode.CONCURRENT)
     @ResourceLock("AbstractSpringIntegrationIndependentTest")
     @AutoConfigureEmbeddedDatabase
-    @ActiveProfiles({ SPRING_PROFILE_TEST, "artemis", PROFILE_SCHEDULING, "lti" })
+    @ActiveProfiles({ SPRING_PROFILE_TEST, PROFILE_ARTEMIS, PROFILE_SCHEDULING, PROFILE_IRIS })
```

Comment:
Did you change this? `lti` and Iris do not seem equal

Reply:
Nope that's the changes coming from develop. Seems like someone made a mistake there